### PR TITLE
Orders

### PR DESCRIPTION
**Pull request by sgfhg12**
### Assignee Tasks

- [ ] added unit tests (or none needed)
- [ ] written relevant docs (or none needed)
- [ ] referenced any relevant issues (or none exist)

### Guidelines

Please add a description of this Pull Request's motivation, scope, outstanding issues or potential alternatives, reasoning behind the current solution, and any other relevant information for posterity.

---

*Your PR Notes Here*

### CODE REVIEW

File: client/components/OrderRow.js [hunk 1]
```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+
+export default class OrderRow extends React.Component {
+    constructor(props){
+        super(props)
+    }
+    render(){
+        console.log(this.props.order)
```
Comment (gabrielmessager) [inline client/components/OrderRow.js]: remove `console.log`

File: client/components/SingleOrder.js [hunk 2]
```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+import { connect } from 'react-redux';
+import { fetchOrder } from '../store/order';
+
+class SingleOrder extends React.Component {
+    constructor(props) {
+        super(props)
+        this.state = {
```
Comment (gabrielmessager) [inline client/components/SingleOrder.js]: are you using `this.state`, anywhere?

File: client/components/SingleOrder.js [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+    }
+    render() {
+        const { order } = this.props
+        return (
+            order ? (
+                <div>
+                    {this.props.order.id}
+
+                </div>
+            ) : null
```
Comment (gabrielmessager) [inline client/components/SingleOrder.js:21]: maybe render a `<h3>Not orders found.</h3>`

File: server/api/orders.js [hunk 4]
```diff
@@ -0,0 +1,16 @@
+const orderRouter = require('express').Router();
+const { Order } = require('../db/models')
+
+orderRouter.get('/:userId', (req, res, next) => {
+    console.log('BODY', req.params);
```
Comment (gabrielmessager) [inline server/api/orders.js]: remove `console.log`

**Review by gabrielmessager — CHANGES_REQUESTED** [review]

**Review by emiliodacosta — DISMISSED** [review]

**Review by gabrielmessager — APPROVED** [review]

**Pull request merged**